src/glob.nim

```diff
@@ -222,7 +222,14 @@ proc splitPattern* (pattern: string): PatternStems =
 
 proc glob* (pattern: string, isDos = isDosDefault): Glob =
   ## Constructs a new `Glob <#Glob>`_ object from the given ``pattern``.
-  let rgx = globToRegexString(pattern, isDos)
+  var rgx = globToRegexString(pattern, isDos)
+
+  # TODO: proper handling: allow user to pass in flag
+  var caseSensitive = true
+  when defined(macosx): # TODO: ospaths.FileSystemCaseSensitive
```

Comment:
why "ospaths.FileSystemCaseSensitive" is todo? :)

Reply:
I think he's referring to https://github.com/nim-lang/Nim/issues/8349